Week2/database.js

```diff
@@ -0,0 +1,112 @@
+const mysql = require('mysql');
+const prompts = require('prompts');
+const { connection, execQuery } = require('./connection');
+const { question1, question2, question3, question4, question5 } = require('./queries');
+
+const getCapitalOfCountry = async () => {
+  try {
+    const response = await prompts({
+      type: 'text',
+      name: 'country',
+      message: `What is the capital of ?`,
+    });
+
+    const capitalQuery = await execQuery(question1, response.country);
+    const capitalOfCountry = capitalQuery[0].capital;
+    console.log(capitalOfCountry);
+  } catch (error) {
+    console.log(`There is no country for this input. ${error.message}`);
+  }
+};
+
+const getLanguagesInRegion = async () => {
+  try {
+    const response = await prompts({
+      type: 'text',
+      name: 'region',
+      message: `List all the languages spoken in the region ...?`,
+    });
+    const userInput = response.region.charAt(0).toUpperCase() + response.region.slice(1);
+    const regionQuery = await execQuery(question2, userInput);
+    const validateQuery = await execQuery(`select region from country group by region`);

```

Comment:
Why this query ? You could just check the length of regionQuery to see if you have language .

Reply:
I thought something different. But you are right.Thank you